dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -183,6 +192,33 @@ private void print(@Nonnull Task<?> task, int indent) {
     }
   }
 
+  private Path prepareOutputPath(
+      @Nonnull String fileName, @Nonnull Closer closer, @Nonnull ConnectorArguments arguments)
```

Comment:
The method accepts a path, not just a file name as a first parameter. I've added more details about not using `String` type in other comments.

Reply:
This is addressed in the other conversations.